.circleci/config.yml

```diff
@@ -76,27 +65,7 @@ workflows:
                 - /^(.*)$/
             branches:
               only:
-                - develop
-                - /release\/.*/
-                - /hotfix\/.*/
-                - feature/circleci
```

Comment:
Since this repository is not git flow-enabled, it would never build on any of these branches. If we want to publish snapshots, then we could add `master` to this list, otherwise it should only publish tagged commits.